membrane_curvature/base.py

```diff
@@ -0,0 +1,72 @@
+import numpy as np
+
+from membrane_curvature.lib.mods import derive_surface, mean_curvature, gaussian_curvature
+import numpy as np
+
+
+def __init__(self, universe, select,
```

Comment:
Do you need `universe` to get passed? Is `select` an AtomGroup or a string -- if it's an AtomGroup, you could just use `universe = select.universe`.

FYI most AnalysisBase classes use `select` as a string, and many accept either an AtomGroup or Universe for the first argument. To safeguard against the multiple possibilities, you could do this:

```python
def __init__(self, universe, select="all", ..., **kwargs):
  super().__init__(universe.universe.trajectory, **kwargs)
  self.selection = universe.select_atoms(select)
```